internal/commands/build.go

```diff
@@ -258,7 +257,7 @@ func buildFlagEnvVar(name string) string {
 var (
 	buildFlagEnvVarPrefix = globalFlagEnvVarPrefix + "BUILD_"
 
-	buildIDFlag = &workaround.IntFlag{
+	buildIDFlag = &cli.StringFlag{
```

Comment:
will this break with an int input or auto parse into string?

Reply:
auto parse into string